app/views/homes/index.html.erb

```diff
@@ -1,3 +1,4 @@
+<div id="app"></div>
 <p>
   Modify me in `app/views/homes/index.html.erb` or change the root directive in
   `config/routes.rb`
```

Comment:
we can go ahead and remove the `<p>` tag and all its contents now that we have the React portion up and running